Packs/BluelivThreatContext/Integrations/BluelivThreatContext/BluelivThreatContext.py

```diff
@@ -0,0 +1,1062 @@
+            if threatActors > 0:
+                threatActorIds = client.get_relationships("tool", str(toolId), "threat-actor")
+
+            human = getHuman(result)
+            demisto.results({
+                'ContentsFormat': formats['json'],
+                'Type': entryTypes['note'],
+                'Contents': result,
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable': tableToMarkdown("Blueliv Tool info", human),
+                'EntryContext': {
+                    'threatContext.hasResults': 'true',
+                    'tool.name': name,
+                    'tool.description': description,
+                    'tool.lastSeen': lastSeen,
+                    'tool.campaigns': campaigns,
+                    'tool.campaignIds': campaignIds,
+                    'tool.signatures': signatures,
+                    'tool.signatureIds': signatureIds,
+                    'tool.threatActors': threatActors,
+                    'tool.threatActorIds': threatActorIds
+                }
+            })
+        else:
+            notFound()
+    else:
+        notFound()
+
+
+def blueliv_signature(client: Client, signatureName, signatureId):
+    if signatureId == '0':
+        signatureId = client.search_by_name('signature', signatureName)
+
+    if signatureId != '0':
+        result = client.get_signature_info(signatureId)
+
+        if result:
+            name = result['data']['attributes']['name']
+            signatureType = result['data']['attributes']['type']
+            updatedAt = result['data']['attributes']['updated_at']
+
+            # MALWARE #
+            malwareIds = ""
+            malware = result['data']['relationships']['malware']['meta']['count']
+            if malware > 0:
+                malwareIds = client.get_relationships("signature", str(signatureId), "malware")
+
+            human = getHuman(result)
+            demisto.results({
+                "Type": entryTypes["note"],
+                'Contents': result,
+                "ContentsFormat": formats["json"],
+                'HumanReadable': tableToMarkdown("Blueliv Signature info", human),
+                'ReadableContentsFormat': formats['markdown'],
+                'EntryContext': {
+                    'threatContext.hasResults': 'true',
+                    'signature.name': name,
+                    'signature.type': signatureType,
+                    'signature.updatedAt': updatedAt,
+                    'signature.malware': malware,
+                    'signature.malwareIds': malwareIds
+                }
+            })
+        else:
+            notFound()
+    else:
+        notFound()
+
+
+# Get inforamtion abouth the provided CVE code
+def blueliv_cve(client: Client, cveCode, vulnId):
+    if vulnId == '0':
+        vulnId = cveCode
+
+    result = client.get_cve_info(vulnId)
+
+    if result:
+        name = result['data']['attributes']['name']
```

Comment:
avoid direct access to dict keys, use the safer way `result.get` with proper default values if needed.